src/openrct2/ride/Ride.cpp

```diff
@@ -7181,7 +7180,6 @@ RideClassification Ride::GetClassification() const
         case RIDE_TYPE_1F:
         case RIDE_TYPE_SHOP:
         case RIDE_TYPE_22:
-        case RIDE_TYPE_50:
```

Comment:
Revert this change and the one on 4353.

Reply:
That's odd, when I look at that file the reference to RIDE_TYPE_50 isn't there.

Reply:
A stray whitespace change.

I'll take care of it.

Reply:
Oh, RIDE_TYPE_50 was removed not added. This should be fixed now.